From 29358a5e3b7a4c8581594b0408ab73f3b13e8e92 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Wed, 16 Nov 2016 16:19:34 +0500
Subject: [PATCH] GpuProfiling: disable ignored-attributes checks

The GCC6 tools introduce a new ignored-attributes type
of warning which is ignorable itself in this case.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 CodeXL/Components/GpuProfiling/Build/Common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CodeXL/Components/GpuProfiling/Build/Common.mk b/CodeXL/Components/GpuProfiling/Build/Common.mk
index 4ab2d12..aaeeb39 100755
--- a/CodeXL/Components/GpuProfiling/Build/Common.mk
+++ b/CodeXL/Components/GpuProfiling/Build/Common.mk
@@ -8,7 +8,7 @@ PLATFORM_CFLAG =
 PLATFORM_LFLAG =
 TARGET_SUFFIX =
 ADDL_CFLAGS = $(CXXFLAGS)
-CFLAGS = $(OPTIMIZE) -std=c++11 -fPIC -Wall -Wno-unknown-pragmas -Wno-strict-aliasing -Wno-non-virtual-dtor -Wno-conversion-null -Werror -msse $(PLATFORM_CFLAG) $(ADDL_CFLAGS)
+CFLAGS = $(OPTIMIZE) -std=c++11 -fPIC -Wall -Wno-unknown-pragmas -Wno-strict-aliasing -Wno-non-virtual-dtor -Wno-conversion-null -Werror -msse -Wno-ignored-attributes $(PLATFORM_CFLAG) $(ADDL_CFLAGS)
 HSAFDN_CFLAGS =
 PLATFORM_DIR = x64
 HSA_PLATFORM_DIR = x86_64
-- 
1.9.1

